pages/userSettings/userSettings.ejs

```diff
@@ -13,6 +13,59 @@
     <div id="content" class="container">
       <h1 class="mb-4">Settings</h1>
 
+      <div class="card mb-4">
+        <div class="card-header bg-primary text-white d-flex align-items-center">
+          Logged-in user profile information
+          <a href="/pl/logout" class="btn btn-danger btn-sm ml-auto">Logout</a>
```

Comment:
Unless there's a specific reason to have a logout button here, let's remove this in favor of the one in the navbar dropdown.